Long term program page

mkaliszewski opened the pull request.

dghazi12 reviewed (commented):
Looks good! Just wanted to point out that some of the words in the navbar are now stacking on top of each other as shown below: 
 
![Screen Shot 2021-04-20 at 11 13 54 PM](https://user-images.githubusercontent.com/59675897/115491503-1afe4980-a22e-11eb-8aa6-a965b4312f53.png)

mkaliszewski commented:
> Looks good! Just wanted to point out that some of the words in the navbar are now stacking on top of each other as shown below:
> 
> ![Screen Shot 2021-04-20 at 11 13 54 PM](https://user-images.githubusercontent.com/59675897/115491503-1afe4980-a22e-11eb-8aa6-a965b4312f53.png)

Thanks for finding that one. I've fixed it.

rishFilet commented:
For me, i am getting underline decoration on the menu items on both the navbar and footer:
![image](https://user-images.githubusercontent.com/28996036/115608102-cf8e7e80-a2b3-11eb-969d-1e987c1070ab.png)
![image](https://user-images.githubusercontent.com/28996036/115608118-d4ebc900-a2b3-11eb-8219-660b0a48796f.png)

rishFilet commented:
The same issue David was describing about the stacking of the menu items is happening for larger screens (above 1440 px).
![image](https://user-images.githubusercontent.com/28996036/115608386-314ee880-a2b4-11eb-9b44-3fc8080a5985.png)

dghazi12 commented:
> The same issue David was describing about the stacking of the menu items is happening for larger screens (above 1440 px).
> ![image](https://user-images.githubusercontent.com/28996036/115608386-314ee880-a2b4-11eb-9b44-3fc8080a5985.png)

This seems to be happening only when the screen size is 1440px now

mkaliszewski commented:
> > The same issue David was describing about the stacking of the menu items is happening for larger screens (above 1440 px).
> > ![image](https://user-images.githubusercontent.com/28996036/115608386-314ee880-a2b4-11eb-9b44-3fc8080a5985.png)
> 
> This seems to be happening only when the screen size is exactly 1440px now on my end

@rishFilet, @dghazi12 Could you pull the newest changes and check how it looks now? I've checked out all resolutions and it seemed fine, but please confirm

![Screenshot from 2021-04-22 08-36-07](https://user-images.githubusercontent.com/42475735/115668050-e7631280-a346-11eb-9ddf-3274b0c2e86d.png)

mkaliszewski commented:
> For me, i am getting underline decoration on the menu items on both the navbar and footer:
> ![image](https://user-images.githubusercontent.com/28996036/115608102-cf8e7e80-a2b3-11eb-969d-1e987c1070ab.png)
> ![image](https://user-images.githubusercontent.com/28996036/115608118-d4ebc900-a2b3-11eb-8219-660b0a48796f.png)
> 
> Also the link for volunteer programs in the footer is going to the landing page.

@rishFilet I've added text-decoration styling to prevent this. Can you let me know what browser are you using? I've tried to run the app on Chrome, Firefox, and Opera and I couldn't reproduce this underline behavior.

dghazi12 commented:
> > > The same issue David was describing about the stacking of the menu items is happening for larger screens (above 1440 px).
> > > ![image](https://user-images.githubusercontent.com/28996036/115608386-314ee880-a2b4-11eb-9b44-3fc8080a5985.png)
> > 
> > 
> > This seems to be happening only when the screen size is exactly 1440px now on my end
> 
> @rishFilet, @dghazi12 Could you pull the newest changes and check how it looks now? I've checked out all resolutions and it seemed fine, but please confirm
> 
> ![Screenshot from 2021-04-22 08-36-07](https://user-images.githubusercontent.com/42475735/115668050-e7631280-a346-11eb-9ddf-3274b0c2e86d.png)


@mkaliszewski So I pulled the updated code, but I am still seeing this on my end. Could just be a weird glitch lol if no one else sees it


![WindaidNavbar](https://user-images.githubusercontent.com/59675897/115813221-cb03bc00-a3c0-11eb-8e59-2056cef1a2f7.gif)

rishFilet commented:
So the text decoration is gone now when i pulled the new code. I am using Firefox on Pop-os (Ubuntu based) on a 27 inch screen.

For the other error i am still seeing it as well, only happening between 1447 and 440 inclusive. Could it be the padding on the left and right of the navbar?

![image](https://user-images.githubusercontent.com/28996036/115882029-58272f00-a41a-11eb-9d67-e99e7fbb523a.png)
![image](https://user-images.githubusercontent.com/28996036/115882143-7b51de80-a41a-11eb-8aba-8883df4fd5c8.png)

mkaliszewski commented:
> So the text decoration is gone now when i pulled the new code. I am using Firefox on Pop-os (Ubuntu based) on a 27 inch screen.
> 
> For the other error i am still seeing it as well, only happening between 1447 and 440 inclusive. Could it be the padding on the left and right of the navbar?
> 
> ![image](https://user-images.githubusercontent.com/28996036/115882029-58272f00-a41a-11eb-9d67-e99e7fbb523a.png)
> ![image](https://user-images.githubusercontent.com/28996036/115882343-ac321380-a41a-11eb-9257-36996d08941d.png)
> 
> ![image](https://user-images.githubusercontent.com/28996036/115882143-7b51de80-a41a-11eb-8aba-8883df4fd5c8.png)


@dghazi12, @rishFilet Unfortunately I still can't reproduce it. 

https://user-images.githubusercontent.com/42475735/116050734-1dbacd80-a678-11eb-8bd4-ecb6d3b41818.mp4



There is still the possibility of having not up to date code, since pulling new changes can sometimes mess up the process.


Please try: 

```
git checkout master
git branch -D mkaliszewski/long-term-program
git fetch
git checkout mkaliszewski/long-term-program
npm run start
``` 

and let me know if this problem is still here

rishFilet commented:
I changed the pixel width in the calc expression to 45px and it works fine now on large screens. I don't see the error. It is strange how it was showing up for David and I and not Mikolaj. @dghazi12 can you confirm with this change you don't see the error anymore? I also was seeing a visual bug in the our core work section.
![image](https://user-images.githubusercontent.com/28996036/116099904-6c696700-a67a-11eb-9a4b-9acf52841800.png)
 
 I removed the width scaling for the md breakpoint on the img div . I don't see that being needed as the responsiveness works fine without it.

mkaliszewski commented:
> I changed the pixel width in the calc expression to 45px for the navbar and it works fine now on large screens. I don't see the error. It is strange how it was showing up for David and I and not Mikolaj. @dghazi12 can you confirm with this change you don't see the error anymore? I also was seeing a visual bug in the our core work section.
> ![image](https://user-images.githubusercontent.com/28996036/116099904-6c696700-a67a-11eb-9a4b-9acf52841800.png)
> 
> I removed the width scaling for the md breakpoint on the img div . I don't see that being needed as the responsiveness works fine without it.

I had to miss that one. Grat job!
